### DISCUSSION TITLE
Why are `FilePondType` and `FilePondCollectionType` separated?

### DISCUSSION THREAD

**Discussion by RafaelKr** [rekalogika/file-src#3] [General]
I was searching for a Symfony FormType providing a good way to integrate FilePond and found this - so much more than expected. On first sight it looks awesome! Thanks for providing this.

But I was asking myself why there is a separation between `FilePondType` and `FilePondCollectionType`? Actually it should be pretty much possible to have one Type which handles both, no?

**Comment by priyadi** [comment]
I don't think there was any particular reason except that it would be simpler to understand. But yes, it should be possible to have one type, that has a branching logic depending on whether multiple is active or not.
